frontends/web/src/components/devices/bitbox02/mnemonicpassphrase.tsx

```diff
@@ -72,13 +72,11 @@ class MnemonicPassphraseButton  extends Component<Props, State> {
                                         {
                                             !mnemonicPassphraseEnabled && message && (
                                                 <p>
-                                                    {
-                                                        message.split('\n').map(line => (
-                                                            <span>
-                                                                <SimpleMarkup tagName="span" markup={line} /><br/>
-                                                            </span>
-                                                        ))
-                                                    }
+                                                    { message.split('\n').map((line, i) => (
+                                                        <span key={i}>
```

Comment:
same here and elsewhere: the array index is not a valid key, as it is not unique per element. totally different entries will have the same keys, which can lead to glitches.

the better key here is just `line` i think.

Reply:
done, using a combination of line + index https://github.com/digitalbitbox/bitbox-wallet-app/compare/e3b8f4422cd2450d8ff969c8440f74501ad9cc2f..932e44070d7fa5168d7a336655892978aca6a6a8